### DISCUSSION TITLE
loading numpy matrices and vectors

### DISCUSSION THREAD

**Discussion by gokhalen** [mfem/mfem#3638] [General]
Is there an easy way to load numpy matrices and vectors into mfem?

-Nachiket

**Comment by sshiraiwa** [comment]
Do you want to do this in the Python wrapper? If so, you could do something like this
```
 v = mfem.Vector(np.array([1,2,3,4.])
 m = DenseMatrix(3,3)
 m.Assign(np.arange(9.).reshape(3,3))
```

**Comment by gokhalen** [comment]
I wanted to do this in the C++ version of MFEM. I think I have managed it using the 'tofile' and 'fromfile' functions from Numpy which appear to write out a simple straight binary file, so I don't need to bother with the npy format
